### ISSUE TITLE
[Bug] Grafana Expression error

### ISSUE THREAD

**Issue by rucciva** [yesoreyeram/grafana-infinity-datasource#295]
**Describe the bug**
Cannot use grafana expression from data returned by the plugin. The log are as follow

```
logger=context t=2022-04-05T08:23:49.83+0000 lvl=eror msg="Query data error" error="expression request error: input data must be a wide series but got type not (input refid)" remote_addr=172.30.0.1
```

**Screenshots**
Before Adding Expression
<img width="1432" alt="Screen Shot 2022-04-05 at 15 22 27" src="https://user-images.githubusercontent.com/4647473/161712885-ed7e30d9-55d6-47e0-b1b5-2a7c66d9213a.png">

After Adding Expression
<img width="1432" alt="Screen Shot 2022-04-05 at 15 14 19" src="https://user-images.githubusercontent.com/4647473/161712892-560030a4-1651-4ec3-9e66-9c0edb1b8dc8.png">

**Version Details:**

- Grafana version : 8.4.5
- Plugin version : 0.8.3

**Additional context**

**Comment by yesoreyeram** [comment]
Expression is not supported by infinity. To support expression this needs to be a full backend datasource.

Curious to know the usecase?

**Comment by rucciva** [comment]
Thanks for the quick response. In this case, a simple math to count success rate with 1 - (err/req). 

Another thing is for alerting that use the new grafana alerting, which heavily relied on reduce and math to fully utilize the new features.

**Comment by yesoreyeram** [comment]
Makes sense. Unfortunately, the plugin doesn't support those backend operations (expressions or alerting).

But in future versions, will consider those. But not sure how feasible without compromising dynamic parsing of json in the frontend.

**Comment by rucciva** [comment]
Well noted. Thanks for considering the feature.

**Comment by rucciva** [comment]
thank you very much @yesoreyeram

**Comment by LaurentDumont** [comment]
Is there any support for the alerting feature with CSV backends? This is really cool!

**Comment by samlaf** [comment]
Would love to see expressions for infinity plugin!
